### PR TITLE
Remove deprecated API and code cleanup.

### PR DESCRIPTION
**Pull request by mfilippov**

### CODE REVIEW

File: src/main/kotlin/com/intellij/ideolog/intentions/LogThisIntentionBase.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      if (visible) {
+        val columnValue = LogParsingUtils.getColumnValueByOffset(editor) ?: "?"
+        lastText = columnValue.toString().trim()
+      }
+
+      return visible
+    }
+
+    override fun invoke(project: Project, editor: Editor, file: PsiFile?) {
+      val set = editor.document.ideologContext.whitelistedItems
```
Comment (knah) [inline src/main/kotlin/com/intellij/ideolog/intentions/LogThisIntentionBase.kt]: Invoke was different between subclasses of this action - different sets are used

Comment (mfilippov) [reply src/main/kotlin/com/intellij/ideolog/intentions/LogThisIntentionBase.kt]: Not sure about it. Set isn't state it is the value extracted from action call.

Comment (knah) [reply src/main/kotlin/com/intellij/ideolog/intentions/LogThisIntentionBase.kt]: I'm fairly certain that `val set = editor.document.ideologContext.hiddenItems` (from LogHideThisIntention.kt) and `val set = editor.document.ideologContext.whitelistedItems` (from LogWhitelistThisIntention.kt) are two different sets.
Looking closely at current state of "hide" and "whitelist" action classes, they have the exact same code (aside from user-visible text), which can't be right.

Comment (mfilippov) [reply src/main/kotlin/com/intellij/ideolog/intentions/LogThisIntentionBase.kt]: I missed it sorry.

**Pull request merged**